lib/file_transfer_agent/file_transfer_agent.js

```diff
@@ -388,15 +388,14 @@ function file_transfer_agent(context)
       {
         if (err)
         {
-          throw(err);
+          //throw(err);
```

Comment:
can you please help me understand why we don't want to throw the error anymore ? what is the error handling instead ?

Reply:
i think the caller should know about if the cleanup of the temp dir was not successful